src/views/__snapshots__/Overview.spec.ts.snap

```diff
@@ -2179,7 +2179,88 @@ exports[`Overview.vue renders basic snapshot 1`] = `
       </div>
        
       <div>
-        <!---->
+        <div
+          class="kong-card card-item border"
+          data-v-0029352b=""
+        >
+          <div
+            class="k-card-header mb-4"
+            data-v-0029352b=""
+          >
+            <div
+              class="k-card-title"
+              data-v-0029352b=""
+            >
+              <h4
+                data-v-0029352b=""
+              >
+                Resources
+              </h4>
+            </div>
+            <div
+              class="k-card-actions"
+              data-v-0029352b=""
+            />
+          </div>
+          <!---->
+          <div
+            class="k-card-body"
+            data-v-0029352b=""
+          >
+            <p
+              data-v-0029352b=""
+            >
+              
+      Join the Kuma community and ask questions:
```

Comment:
where is this defined?

Reply:
In `src/components/Resources.vue:8`. The string is partially dynamic.